bundle/downstream/manifests/quay-bridge-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,513 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    olm.skipRange: ">=3.5.x <3.6.0"
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "quay.redhat.com/v1",
+          "kind": "QuayIntegration",
+          "metadata": {
+            "name": "quay"
+          },
+          "spec": {
+            "clusterID": "openshift",
+            "credentialsSecret": {
+              "name": "quay-credentials",
+              "namespace": "openshift-operators"
+            },
+            "insecureRegistry": false
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: OpenShift Optional
+    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-bridge-operator-rhel8:v3.6.0
+    createdAt: 2021-07-09 14:07 UTC
```

Comment:
@harishsurf Can you confirm this is replaced during the "build" process?

Reply:
yes, the date gets updated using [this line from the Makefile](https://dist-git.host.prod.eng.bos.redhat.com/cgit/containers/quay-bridge-operator-bundle/tree/Makefile?h=quay-3.6-rhel-8#n4)